.travis.yml

```diff
@@ -1,9 +1,12 @@
 # Set Ruby as the language so it doesn't download the pip things. Instead, let docker do that.
 language: ruby
 cache: bundler
+install:
+  - docker-compose -f travis-docker-compose.yml build
```

Comment:
What does this do? Is it done automatically before the script section but not before the install section?